scripts/env.sh

```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
 USER_ID=`id -u $USER`
 USER_NAME="user"
 
+if [[ "${1}" =~ "create_cita_config" ]]; then
+    CONTAINER_NAME="cita_config${RELEASE_DIR//\//_}"
```

Comment:
`"cita_config"` and `"cita_run"` should be a variable before define `CONTAINER_NAME`.
So, we can write `${RELEASE_DIR//\//_}` only once.

Reply:
:clap: